docs/26_export_manual.md

```diff
@@ -41,8 +41,9 @@ will be produced.
 
 #### m_selected_contest_audit_details_by_cvr.sql
 
- For each contest under audit, and for each cast vote record presented to the 
-  Audit Board for verification that contains the given contest, 
+ For each contest under audit, and for each cast vote record in the random sequence
+ (with duplicates) that contains the given contest and has been presented to the 
```

Comment:
This is no longer true, which is why I removed talk of dups in this table. We provide the necessary information via the counted column. They have to figure out dups via the cvr_ids, if they want.